rules/ruleset.go

```diff
@@ -632,6 +646,29 @@ func (rs *ruleSet) Schema() (*schema.RuleSet, error) {
 	return res, nil
 }
 
+func (rs *ruleSet) MappingRules() MappingRules {
+	mappingRules := make(MappingRules, len(rs.mappingRules))
+	for _, m := range rs.mappingRules {
+		mappingRules[m.uuid] = m.history()
+	}
+	return mappingRules
+}
+
+func (rs *ruleSet) RollupRules() RollupRules {
+	rollupRules := make(RollupRules, len(rs.rollupRules))
+	for _, r := range rs.rollupRules {
+		rollupRules[r.uuid] = r.history()
+	}
+	return rollupRules
+}
+
+//UpdateMetadata contains descriptive information that needs to be updated
```

Comment:
nit: space after //

Reply:
also move this outside of the sections where you define all the ruleSet methods? maybe closer to its constructor?

Reply:
Will do